GameDataParser/Parsers/SkillParser.cs

```diff
@@ -35,7 +38,10 @@ protected override List<SkillMetadata> Parse()
                         int levelValue = level.Attributes["value"].Value != null ? int.Parse(level.Attributes["value"].Value) : 0;
                         int spirit = level.SelectSingleNode("consume/stat").Attributes["sp"] != null ? int.Parse(level.SelectSingleNode("consume/stat").Attributes["sp"].Value) : 0;
                         float damageRate = level.SelectSingleNode("motion/attack/damageProperty") != null ? float.Parse(level.SelectSingleNode("motion/attack/damageProperty").Attributes["rate"].Value) : 0;
-                        skillLevels.Add(new SkillLevel(levelValue, spirit, damageRate, feature));
+                        string sequenceName = level.SelectSingleNode("motion/motionProperty") != null ? level.SelectSingleNode("motion/motionProperty").Attributes["sequenceName"].Value : "";
+                        string motionEffect = level.SelectSingleNode("motion/motionProperty") != null ? level.SelectSingleNode("motion/motionProperty").Attributes["motionEffect"].Value : "";
```

Comment:
This is selecting the same node in both instances, should be moved to a separate variable